pkg/config/testdata/test_other_yamls/global-project.yaml

```diff
@@ -15,3 +15,4 @@ spec:
   - name: project-admins
     type: Group
     role: ProjectAdmin
+    dn: cn=Developers,ou=users,dc=example,dc=com
```

Comment:
lol, I fixed this bug by removing the whole `project-admins`

Reply:
will be fixed after master merge